legacy/messaging/src/test/java/org/jboss/as/messaging/test/MigrateTestCase.java

```diff
@@ -152,19 +152,21 @@ private void testMigrateOperation(boolean addLegacyEntries) throws Exception {
         migrateOp.get(OP_ADDR).add(SUBSYSTEM, SUBSYSTEM_NAME);
 
         ModelNode response = services.executeOperation(migrateOp);
-
+        System.out.println("Response " + response);
         checkOutcome(response);
 
         ModelNode warnings = response.get(RESULT, "migration-warnings");
         // 6 warnings about broadcast-group attributes that can not be migrated.
         // 2 warnings about broadcast-group attributes not migrated because they have an expression.
+        // 3 warnings about broadcast-group not migrated because they don't have a proper network configuration.
```

Comment:
Which are the 3 broadcast-groups that can not be migrated?

There is `groupA` and `groupB` (that uses `group-address` & `group-port` attributes) but `groupS` and `groupT` looks fine.